types/node/events.d.ts

```diff
@@ -154,7 +154,11 @@ declare module 'events' {
          * ```
          * @since v11.13.0, v10.16.0
          */
-        static once(emitter: NodeEventTarget, eventName: string | symbol, options?: StaticEventEmitterOptions): Promise<any[]>;
+        static once(
```

Comment:
Please refrain from making unrelated changes.

Reply:
Actually, that change is made by prettier, if these are not fixed, every contributor will have to manually revert every style fixes which adds a burden on contributing, I can revert if that's what you end deciding.